app/assets/javascripts/repositories/index.js.erb

```diff
@@ -0,0 +1,5 @@
+$(document).ready(function() {
+  $('.repository-nav-tab').each(function() {
+    truncateLongString($(this), <%= Constants::NAME_TRUNCATION_LENGTH %>);
+  });
+})
```

Comment:
You don't need this file because you don't need to do this dynamically - you can call `truncateLongString` in HTML on the value of `a` tag.